README.adoc

```diff
@@ -123,130 +124,52 @@ include::{common-includes}/gitclone.adoc[]
 
 // no "try what you'll build" section in this guide because it would be too long due to all setup the user will have to do.
 
-// =================================================================================================
-// Managing an Azure Container Registry
-// =================================================================================================
-
-== Managing an Azure Container Registry
-
-To deploy your microservices, you need to create an Azure Container Registry in the same location 
-where your services are deployed, and link the registry to a resource group. Your registry will manage container instances 
-that will be deployed to a Kubernetes cluster.
+== Provision deployment foundation with IBM Liberty on AKS 
 
 === Creating a resource group
 
+A resource group is an Azure construct to manage a logical collection of resources for your cloud deployments on Azure. You must create a new resource group to manage the resources you need for your Kubernetes deployment.
 
-A resource group is an Azure construct to manage a logical collection of resources for your cloud deployments on Azure. 
-You must create a new resource group to manage the resources you need for your Kubernetes deployment.
+To create the resource group in an efficient way, check out the https://aka.ms/liberty-aks[IBM Liberty on AKS] tool and click on the `Create` button to start the process. 
```

Comment:
"...[IBM Liberty on AKS] tool ..." -> "...[IBM WebSphere Liberty and Open Liberty on AKS] plan..."